frontend/src/components/form/ServerErrorContent.vue

```diff
@@ -2,15 +2,15 @@
   <div v-if="!!serverError.name || !!serverError.response ">
     <template v-if="serverError.name === 'ServerException' && serverError.response && serverError.response.status === 422">
       <div class="title">
-        Validation error:
-      </div>
-      <div v-for="(validation_messages, name) in serverError.response.data.validation_messages" :key="name">
-        <div>
-          <b>{{ name }}</b>: <span v-for="(message, idx) in validation_messages" :key="idx">
-            {{ message }}<span v-if="idx === validation_messages.length">, </span>
-          </span>
-        </div>
+        Validation error
```

Comment:
Gleich übersetzbar machen?

Reply:
Not sure. Die Server-Errors kommen im Moment sowieso alle in Englisch, dann hätten wir da einen komischen mix aus Deutsch und Englisch. 
Die Hoffnung wäre, dass dies dem User nie bzw. sehr selten angezeigt werden muss...

Reply:
Ja gut ich würde hoffen dass wir die üblichen Fehler (Validierung) früher oder später anhand der Informationen aus dem Backend übersetzen können:
```json
{
  "propertyPath":"email",
  "message":"This value should not be blank.",
  "code":"c1051bb4-d103-4f74-8988-acbcafc7fdc3"
}
```
Aber das ist ja dann noch ein gröberer Effort bis alle Entity- und Feldnamen und die üblichen Validierungen (identifiziert durch den `"code"`) übersetzt sind. Diesen String "Validation error" können wir auch dann noch übersetzen.

Reply:
Oder direkt im Backend übersetzen?
Für die Standard-Validators scheint es bereits Übersetzungen zu geben:
https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Validator/Resources/translations/validators.de.xlf#L89

Reply:
Ja, könnte man vermutlich mit einem `Accept-Language` Header auch RESTful stateless und cache-bar umsetzen. Aber den `propertyPath` würde ich im Frontend auch "übersetzt" anzeigen wollen, z.B. "E-Mail: Dieser Wert darf nicht leer sein" oder "Periode 1 &rarr; Ende: Darf nicht früher als 01.07.2022 sein". Und spätestens dieses Mapping von "." auf "&rarr;" und von "[0]" auf " 1" ist reine Kosmetik die in unterschiedlichen API clients unterschiedlich gelöst werden könnte: Oder vielleicht entscheidet ja ein API Client, statt "Periode 1" den eingegebenen Namen der Periode anzuzeigen: "Vorweekend &rarr; Ende: Darf nicht früher als 01.07.2022 sein"
Daher bin ich fast sicher dass das ins Frontend gehört, bzw. der `propertyPath: "periods[0].end"` muss sicher als technisch einfach parsebaren "Key" verfügbar bleiben. Und bevor wir da zusätzliche Properties ala `translatedPropertyPath` einfügen, würde ich noch kurz schauen ob da Symfony nicht einem Standard folgt, und ob das JSON in allen unterstützten [error Formaten](https://api-platform.com/docs/core/content-negotiation/#configuring-error-formats) gleich aussieht.